main.tf

```diff
@@ -54,6 +54,7 @@ module "storage" {
   tags = local.common_labels
 
   depends_on = [module.aks]
+  subnets    = [module.aks.subnet_id]
```

Comment:
So ... I actually reran with `depends_on = [module.aks.subnet_id]` instead of `depends_on = [module.aks]` -- since with  `depends_on = [module.aks.subnet_id]` , the `module.aks` is the implicit dependency.  That seems to take care of the timing issue.

Reply:
Cool, I'll make this fix and re-push. Thanks!!